pkg/koordlet/metriccache/storage_test.go

```diff
@@ -848,6 +856,14 @@ func Test_storage_ContainerPSIMetric_CRUD(t *testing.T) {
 				}
 			}
 
+			gotNum, err := s.CountContainerPSIMetric()
```

Comment:
To count the metrics individually, we may create the metric cache via `newStorage()` while the `NewStorage` would open the same DB.

Reply:
both are the same db, and needs defer close now.